### ISSUE TITLE
Error no shared text when share photo in iphone

### ISSUE THREAD

**Issue by pcorqui** [CrossGeeks/FacebookClientPlugin#80]
If you are creating an issue for a BUG please fill out this information. If you are asking a question or requesting a feature you can delete the sections below. 

**Failure to fill out this information will result in this issue being closed.** If you post a full stack trace in a bug it will be closed, please post it to http://gist.github.com and then post the link here.

## Bug Information
Error when shared photo with text in iOS

Version Number of Plugin: 2.2.1
Device Tested On: iphone 8 and ipone 11
Simulator Tested On:
Version of VS: 16.6.0
Version of Xamarin: 4.6.0.847
Versions of other things you are using:

### Actual Behavior
in android is fine, but in iOS only shared the photo whitout text.

### Code snippet

FacebookSharePhoto photo = new FacebookSharePhoto(text, data);
                        FacebookSharePhoto[] photos = new FacebookSharePhoto[] { photo };
                        FacebookSharePhotoContent photoContent = new FacebookSharePhotoContent(photos, text);
                        await CrossFacebookClient.Current.ShareAsync(photoContent);

**Comment by DeanZhuo** [comment]
In my case, the 2nd parameter is an URL, so I used null as my input, and write the text at the third parameter.
FacebookSharePhotoContent photoContent = new FacebookSharePhotoContent(photos, null, Message);

**Comment by pcorqui** [comment]
Yeah, I have try this option in android it's functionality is correct but in iOS no share the text. in this moment I cant debug in iOS but I try debug and check the solution
